pandas/core/window.py

```diff
@@ -996,6 +1007,17 @@ def _get_corr(a, b):
 
 
 class Rolling(_Rolling_and_Expanding):
+    _attributes = ['window', 'min_periods', 'freq', 'center', 'win_type',
+                   'axis', 'on', 'closed']
+
+    def __init__(self, obj, window=None, min_periods=None, freq=None,
+                 center=False, win_type=None, axis=0, on=None,
+                 closed='right', **kwargs):
+        self.closed = closed
+        super(Rolling, self).__init__(obj=obj, window=window,
+                                      min_periods=min_periods, freq=freq,
+                                      center=center, win_type=win_type,
```

Comment:
Should I then move `self.closed = closed` and the addition to `_attributes` to `_Window.__init__`?